### PR TITLE
STORM-749: Remove CSRF check.

### PR DESCRIPTION
**Pull request by Parth-Brahmbhatt**

### CODE REVIEW

**Comment by harshach** [comment]
+1

**Comment by HeartSaVioR** [comment]
LGTM.
Maybe it would be better to remove "antiForgeryToken" from /api/v1/topology/:id's sample json.

File: storm-core/src/clj/backtype/storm/ui/core.clj [hunk 1]
```diff
@@ -975,7 +971,7 @@
             name (.get_name tplg)]
         (.activate nimbus name)
         (log-message "Activating topology '" name "'")))
-    (json-response (topology-op-response id "deactivate") (m "callback")))
+    (json-response (topology-op-response id "activate") (m "callback")))
```
Comment (d2r) [inline storm-core/src/clj/backtype/storm/ui/core.clj]: Good catch.  This needs to be fixed in 0.10.x-branch also.

**Comment by d2r** [comment]
> Maybe it would be better to remove "antiForgeryToken" from /api/v1/topology/:id's sample json.

Yes, STORM-UI-REST-API.md line 379.

**Comment by d2r** [comment]
https://github.com/Parth-Brahmbhatt/incubator-storm/blob/b7ed77727a1d1e76fc775ac6776b6580e25a1afe/storm-core/src/ui/public/topology.html#L125-L128

Should remove this too, since the template is gone.

**Comment by Parth-Brahmbhatt** [comment]
@d2r @HeartSaVioR fixed.

**Comment by d2r** [comment]
Thanks.  I am +1 for this when we have a similar pull request for the 0.10.x-branch.

**Comment by HeartSaVioR** [comment]
I have a question. When we want to reflect PR into master and existing branch, is it encouraged to make two PRs pointing for each branches, or just one PR with cherry-pick?

**Comment by d2r** [comment]
> I have a question. When we want to reflect PR into master and existing branch, is it encouraged to make two PRs pointing for each branches, or just one PR with cherry-pick?

I think we could cherry-pick, but either way you would have a new branch before the code is merged in.

1) check out 0.10.x-branch
2) create new branch
3) cherry-pick change (test, etc...)
4) make pull request to apache 0.10.x-branch

I do not think we should cherry-pick commits directly to a branch without a pull request, because it is a code change that should have a review.

@ptgoetz, any comment?

**Comment by Parth-Brahmbhatt** [comment]
Here is Pr against 0.10-x branch https://github.com/apache/storm/pull/512

**Comment by d2r** [comment]
+1

**Pull request merged**